docs/content/manual/_incoming/2906-vm-boot-stress-test .md

```diff
@@ -0,0 +1,40 @@
+---
+title: VM boot stress test 
+---
+
+* Related issues: [#2906](https://github.com/harvester/harvester/issues/2906) [BUG] VM can’t boot due to filesystem corruption
+
+  
+## Category: 
+* Volume
+
+## Verification Steps
+1. Create volume (Harvester, Longhorn storage class)
+1. Create volume from image
+1. Unmount volume from VM
+1. Delete volume in use and not in use
+1. Export volume to image 
+1. Create VM from the exported image
+1. Edit volume to increase size 
+1. Delete volume in use 
+1. Clone volume 
+1. Take volume snapshot
+1. Restore volume snapshot 
+1. Utilize the E2E test in harvester/test repo to prepare a script to continues run step 1-11 at lease 100 runs
+
+
+## Expected Results
+1. Pass more than 300 rounds of the I/O write test, should encounter data corruption issue and VM is alive
```

Comment:
the expected result should NOT encounter data corruption, otherwise the bug will still exist.

Reply:
Thanks for the check, change to **Should Not** encounter data corruption issue in #2906